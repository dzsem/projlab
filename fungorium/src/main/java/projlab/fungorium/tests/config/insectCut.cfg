# Insect Cut
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD TECTON
REGISTER NEIGHBOUR 0 1
# 2
ADD MUSHROOMTHREAD 0 0
# 3
ADD MUSHROOMTHREAD 0 1
REGISTER CONNECTION 2 3
# 4
ADD INSECT 1 0