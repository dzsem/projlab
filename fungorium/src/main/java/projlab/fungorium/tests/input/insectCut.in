# Insect Cut
# ------------------------------------------
# Input
# ------------------------------------------
INSECT 4 CUTTHREAD 2
# Print states
# ------------------------------------------
LIST MUSHROOMTHREAD