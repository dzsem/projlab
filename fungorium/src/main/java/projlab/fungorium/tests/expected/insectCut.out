MUSHROOMTHREAD 2 0 0 1 CUT SPROUT
MUSHROOMTHREAD 3 0 1 1 UNCUT SPROUT